dub.json

```diff
@@ -43,6 +43,16 @@
             "targetName": "agora",
             "mainSourceFile": "source/agora/node/main.d",
             "debugVersions": [ "AgoraDebugAPIs" ],
```

Comment:
Maybe remove the debug API too ?

Reply:
I would rather keep it during the testnet, in case we run in to a consensus problem.